test/dummy/config/initializers/secret_token.rb

```diff
@@ -4,5 +4,12 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-Dummy::Application.config.secret_token    = 'deadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeefdeadbeef'
+#
+# avoid deprecation warnings if building against older versions of Rails
+SKB_VERSION = Gem::Version.new('4.0.0')
+RG_VERSION = Rails.respond_to?(:version) ? Gem::Version.new(Rails.version) : Gem::Version.new('3.22')
```

Comment:
[QUESTION] What does "SKB" and and "RG" stand for?  Unless I'm being dumb here, could you please rename these constants?

Reply:
SKB is shorthand for "secret_key_base" -- basically, the idea is to set secret_token only if you're running an old version of the Rails Gem (RG).  I updated the comments in my branch to make this more explicit.